extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -36,6 +36,21 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', [
 			addEventListener: function (eventName, callback) {
 				ima.addEventListener(eventName, callback);
 			},
+			computeVastMediaAspectRatio: function () {
+				var aspectRatio = width / height,
+					currentAd;
+
+				if (ima.getAdsManager()) {
+					currentAd = ima.getAdsManager().getCurrentAd();
+					aspectRatio = currentAd.getVastMediaWidth() / currentAd.getVastMediaHeight();
+
+					if (!aspectRatio) {
```

Comment:
I've talked with people in the team how `getVastMediaWidth` and `getVastMediaHeight` behave and it might return 0 even if creative is selected.